ClickableTransparentOverlay/ImGuiController.cs

```diff
@@ -310,6 +310,7 @@ public void Render(GraphicsDevice gd, CommandList cl)
             if (_frameBegun)
             {
                 _frameBegun = false;
+                ImGui.EndFrame();
```

Comment:
Was it a bug? Why was ImGui working fine without adding this before?

Reply:
FYI: // This is normally called by Render(). You may want to call it directly if you want to avoid calling Render() but the gain will be very minimal.

Ref: https://github.com/ocornut/imgui/blob/master/imgui.cpp#L4205

Reply:
No you're right it wasn't a bug. I had issues while debugging inputs and was trying different things out and forgot to revert this